### PR TITLE
DDF-4467 Validating lat/lon coords

### PR DESCRIPTION
**Pull request by gjvera**
#### What does this PR do?
Adds validation that the user can see before executing a query. If a user tries to input a lat/lon coordinate that is invalid, then the UI will not let them do so.
#### Who is reviewing it? 
@mojogitoverhere 
@willwill96 
@rymach 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 
@djblue 
#### How should this be tested?
Create an advanced or basic query
create an anyGeo filter, with `point radius` and `bounding box` filters
enter invalid coordinates and ensure the UI will not let you 
ensure no regressions in the geospatial search

#### What are the relevant tickets?
[DDF-4467](https://codice.atlassian.net/browse/DDF-4467)
#### Screenshots
fix

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js [hunk 1]
```diff
@@ -46,18 +46,20 @@ function convertToValid(key, model) {
       parseFloat(key.mapSouth || model.get('mapSouth')) + minimumDifference
   }
   if (key.mapNorth !== undefined) {
-    key.mapNorth = Math.max(-90 + minimumDifference, key.mapNorth)
+    key.mapNorth = Math.max(-90, key.mapNorth)
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js]: I'm guessing the minimumDifference logic is no longer necessary given all the changes we've made in this area over the years.  Might be worth removing all of the references.  Although make sure to test all the location input styles and verify that there are no weird issues at the extremes.

**Comment by andrewkfiedler** [comment]
How does this look at the moment?  Could you post a video if you have a chance?

**Comment by gjvera** [comment]
@andrewkfiedler added

**Comment by mojogitoverhere** [comment]
build now

**Comment by gjvera** [comment]
@SmithJosh good catch. Fixed.

**Comment by willwill96** [comment]
🌮 🌮 Hero build successful 🌮 🌮 
1. Quickbuilt and installed
2. Created searches with bounding box and point-radius and tried entering in invalid coordinates - verified that I was unable to

Note: also verified that I was unable to enter the coordinates that @SmithJosh mentioned

**Comment by gjvera** [comment]
build now

**Comment by gjvera** [comment]
build now

**Pull request merged**